### ISSUE TITLE
Emoji like on github

### ISSUE THREAD

**Issue by vetash** [RocketChat/Rocket.Chat#1838]
How i can change these flat emoji on emoji like in slack?

**Comment by marceloschmidt** [comment]
Right now, you can't. There's an open issue for custom emojis already: #1723.

**Comment by MatthewFellows** [comment]
I could be wrong, but I get the impression he's asking to change the whole emjoi icon set to another one.  Is this possible already?

**Comment by Schwoebel** [comment]
I echo what MatthewFellows is saying, I think this ticket is for changing emoji sets.  Not for just adding custom emoji's like #1723 is talking about.

**Comment by lucasvanhalst** [comment]
With the PR #4167 from @nokobon I think this will be easier to implement, although I'm not entirely sure.

**Comment by nokobon** [comment]
Yes, it would be easier to implement with how I added custom emoji. Different sets of emoji could be added by creating other packages, but there would also need to be a setting or something to enable/disable emoji sets.

**Comment by Schwoebel** [comment]
I think @marceloschmidt should consider re-opening this.

**Comment by Schwoebel** [comment]
Damn that was quick. Sorry I can't give your reopening this a thumbs up!

**Comment by JasinYip** [comment]
We've use Rocket.Chat to instead of Slack because of Slack's expensive price yesterday, and we are looking for the change that we could change the emoji set to make it looks like Github's...😂

So any news here?

**Comment by rybak-g** [comment]
That would be great indeed, anyway to reopen the ticket ?

**Comment by tadeu** [comment]
The +1 (:+1:) emoji is SO UGLY!

![image](https://user-images.githubusercontent.com/955251/32052380-d420dd9e-ba36-11e7-98b9-e3a43ba115d6.png)

Here are some alternatives for comparison: https://emojipedia.org/thumbs-up-sign/

**Comment by coding-horror** [comment]
Please, PLEASE let us select the emoji set in use! I have zero desire (or need) to set custom emoji but I really want to pick from Apple, Google, Twitter, EmojiOne, Microsoft emoji sets globally for our rocket.chat instance!

**Comment by SamSaffron** [comment]
Note, at Discourse we ship with 6 sets and allow users to pick which set they want. 

Totally happy for you to borrow any of the code we have that is used for updating the sets which is the bulk of the work when adding more sets.

**Comment by Otiel** [comment]
Any chance to see this feature implemented soon?

**Comment by stuartpb** [comment]
@SamSaffron Would you be able to point to the code one would borrow if they were inclined to reuse Discourse's work?

**Comment by igalic** [comment]
github uses a heuristic to determine whether a platform already has native emoji support… so, running on Linux, i fake it and tell GitHub that i'm on Safari, so it'll just use Noto Color Emoji, which almost every distro ships.

I don't quite understand the need to ship one's own Emoji set, let alone 6 different ones.

**Comment by MattFellows** [comment]
@igalic The use case is that sometimes the emoji sets that ship with an OS or distro are fugly, in a lot of people's opinions, even the ones that are built into RocketChat have some questionable images.  Having the option to override that with a custom set, e.g. google's, or one's you've designed yourself, is a nice thing to have.

**Comment by igalic** [comment]
if we're adding all these options, perhaps it would also be nice to add an option for `user system fonts / system emoji`

**Comment by guidozuc** [comment]
Any update on the ETA for this feature?
